src/main/resources/JavadocParser.g4

```diff
@@ -48,15 +48,15 @@ linkPlainInlineTag: LINKPLAIN_LITERAL reference description?;
 linkInlineTag: LINK_LITERAL reference description?;
 
 reference
-        : qualifiedName (HASH memberReference)?
+        : (qualifiedName SLASH)? qualifiedName (HASH memberReference)?
```

Comment:
So we are going to have 2 qualified names in this `reference` and not identify them separately like we did `memberReference` and it's children of `parameterTypeList`?